package.json

```diff
@@ -6,7 +6,7 @@
 		"build": "next build",
 		"export": "next export",
 		"start": "next start",
-		"test": "jest --env=jsdom"
+		"test": "jest --env=jsdom --watchAll"
```

Comment:
we should create a new command, `test:dev` or something, that does the watch, because if we do this then when we add a hook to run the test it will run endlessly.